docs/get_started/getting_started_riscv_cmake.md

```diff
@@ -35,21 +29,37 @@ $ ./build_tools/riscv/riscv_bootstrap.sh
 
 ## Configure and build
 
-### Configure on Linux
+### Host configuration
+
+Build and install at least the compiler tools on your host machine, or install them from a binary distribution:
+
+```shell
+$ mkdir build-host ; cd build-host ; mkdir install
+$ cmake -G Ninja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=./install ..
+$ cmake --build . --target install
```

Comment:
nit: I notice for Android, the host compiler is built outside of the iree root directory. Maybe for the sake of consistency, the shell script here can be the same? The host compiler should be shareable between Android and RiscV targets.